### DISCUSSION TITLE
Feedback on popups in Search Results template

### DISCUSSION THREAD

**Discussion by meckert-pax8** [microsoft-search/pnp-modern-search#2137] [Q&A]
I wanted to ask for feedback if its possible through custom results template to have a popup in the search results that will display a PowerApps form either in iframe or linked directly, with query string parameters.  Open to ideas to accomplish this task, thank you

**Comment by wobba** [comment]
Yes, if you develop it as a web component. See https://microsoft-search.github.io/pnp-modern-search/extensibility/custom_web_component/
